server.js

```diff
@@ -1,11 +1,135 @@
 const express = require('express')
 const app = express()
+const cors = require('cors')
+const bodyParser = require('body-parser')
+const { Dvr } = require('@mui/icons-material')
 const port = 5000
 
+const sqlite3 = require('sqlite3').verbose();
+
+const db = new sqlite3.Database('./shopListDB.db', sqlite3.OPEN_READWRITE, (err) => {
+  if(err) 
+    return console.error(err.message);
```

Comment:
`console.error` doesn't return anything
```suggestion
    console.error(err.message);
    return
```